examples/lambda/function.yaml

```
@@ -2,31 +2,38 @@ apiVersion: lambda.aws.upbound.io/v1beta1
 kind: Function
 metadata:
   annotations:
-    meta.upbound.io/example-id: connect/v1beta1/function
+    meta.upbound.io/example-id: lambda/v1beta1/function
```

```suggestion
    meta.upbound.io/example-id: lambda/v1beta1/function
    uptest.upbound.io/timeout: "3600"
```

Can you add the above as it may take 45m to delete the Subnet?